File: week-03/day-3/07.py
# create a function that takes a list and returns a new list with all the elements doubled

def doubled(list1):
    for i in range(len(list1)):
        list1[i] *= 2
    return list1

list2 = [1, 2, 3, 4, 5, 6, 7, 8, 9]
print(doubled(list2))

File: week-07/day-3/07.js
'use strict';

var numbers = [2, , 11, 29, 67280421310721];

// create a function that takes an array of numbers and returns a boolean
// it should return true if all the elements are prime, false otherwise

function isPrimeArray(arr) {

    function itsPrime(number) {
    for( var j = 2; j < number; j++ ) {
      if( number % j === 0 ) {
        return false;
      }
    }
    return true;
  }
  return arr.every(itsPrime)
}

console.log(isPrimeArray(numbers));
